### PR TITLE
343 add secondary modifier for button

### PR DESCRIPTION
**Pull request by thiagoabreudev**
# Related issue
[343](https://github.com/DivanteLtd/storefront-ui/issues/343)
# Scope of work
added sf-button modifier - outline

# Screenshots of visual changes
outline-buttons

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/component-rules.html) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/SfButton.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import '../variables';
+@import '../helpers';
```
Comment (leomp12) [inline packages/shared/styles/components/SfButton.scss]: Can't include helpers here, it's imported on global scss file and would be duplicated..

File: packages/shared/styles/components/SfButton.scss [hunk 2]
```diff
@@ -9,27 +10,49 @@ $button-font-size: $font-size-small-mobile !default;
...
-    font-size: $button-font-size-desktop;
-  }
-  &:hover {
-    background-color: lighten($button-background-color, 5);
-  }
-  &--full-width {
-    width: 100%;
-  }
+.sf-button {
+	padding: $button-padding;
```
Comment (leomp12) [inline packages/shared/styles/components/SfButton.scss]: Can you fix indentation back to two spaces (as the rest of our source)?

File: packages/shared/styles/components/SfButton.scss [hunk 3]
```diff
@@ -32,4 +33,25 @@ $button-line-height: 1.6 !default;
   &--full-width {
     width: 100%;
   }
+  &--outline {
+    color: $c-black;
+    border: solid $button-outline-border $c-black;
+    background-color: transparent !important;
+    &:hover {
+      background-color: lighten($c-black, 5) !important;
```
Comment (przspa) [inline packages/shared/styles/components/SfButton.scss]:
```suggestion
      background-color: lighten($c-green-primary, 5) !important;
```

File: packages/shared/styles/components/SfButton.scss [hunk 4]
```diff
@@ -32,4 +33,25 @@ $button-line-height: 1.6 !default;
   &--full-width {
     width: 100%;
   }
+  &--outline {
+    color: $c-black;
+    border: solid $button-outline-border $c-black;
```
Comment (przspa) [inline packages/shared/styles/components/SfButton.scss]:
```suggestion
    border: solid $button-outline-border $c-green-primary;
```

File: packages/shared/styles/components/SfButton.scss (hunk 3 above)
Comment (thiagoabreudev) [reply packages/shared/styles/components/SfButton.scss]: Thank you so much for your suggestion!

**Pull request merged**